### PR TITLE
Add pluginval v0.3.0

### PR DESCRIPTION
**Pull request by VirtualAnalogy**
Adding cask for pluginval.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/pluginval.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cask "pluginval" do
+  version "0.3.0"
+  sha256 "1678f91c1afed6857a7a7c05bb13fc2b5586a7a29b02381751aa622c0386429c"
+
+  url "https://github.com/Tracktion/pluginval/releases/download/v#{version}/pluginval_macOS.zip",
+      verified: "https://github.com/Tracktion/pluginval/"
+  name "pluginval"
+  desc "Is a cross-platform plugin validator and tester application"
```
Comment (neutric) [inline Casks/pluginval.rb:8]: `desc "Cross-platform audio plugin validator and tester application"`

File: Casks/pluginval.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  homepage "https://www.tracktion.com/develop/pluginval"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```
Comment (neutric) [inline Casks/pluginval.rb:13]: `url :stable` is not necessary, see e.g. [`openra.rb`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/openra.rb#L12) or [`shield.rb`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/shield.rb#L12).

File: Casks/pluginval.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  app "pluginval.app"
+
+  zap trash: [
+    "~/Library/Caches/pluginval/pluginval_crash.txt",
+    "~/Library/Application\ Support/pluginval/pluginval.xml",
```
Comment (neutric) [inline Casks/pluginval.rb:20]: Please sort the `zap` entries alphabetically. Also, `.../Application Support/...` without `\`.

**Review by neutric — COMMENTED** [review]
Thanks for your first contribution to `homebrew-cask`, @VirtualAnalogy! My review is in the code. Take it with a grain of salt as I am not a maintainer.

**Comment by miccal** [comment]
Thank you @VirtualAnalogy.

**Pull request merged**